docs/static/getting-started-with-logstash.asciidoc

```diff
@@ -41,6 +41,9 @@ automatically detect your environment and install the correct startup method
 JAVA_HOME environment variable during package installation time, you may get an
 error message, and Logstash will be unable to start properly.
 
+IMPORTANT: Users have reported issues with Debian or RPM install packages and Java 11. 
+We are investigating.
```

Comment:
can you add a link to the related issue: https://github.com/elastic/logstash/issues/10593